### PR TITLE
add metabot animation

### PR DESCRIPTION
**Pull request by alxnddr**
[Product doc](https://www.notion.so/metabase/Allow-EE-customers-to-override-our-fun-product-voice-7788c18b79d84bed828fde40d408e26d)

## Changes

Adds Metabot show/hide setting animation.

## How to verify

- Run Metabase EE
- Go to http://localhost:3000/admin/settings/whitelabel
- Change Metabot visibility setting and ensure it animates the change

<video src="https://user-images.githubusercontent.com/14301985/174678186-e5e3e198-8355-4009-a99b-b0d0b3e08aa5.mov" />

### CODE REVIEW

**Review by kulyk — APPROVED** [review]
It's as beautiful as it's sad

**Review by WiNloSt — APPROVED** [review]

File: enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabotSettingWidget/MetabotSettingWidget.tsx [hunk 1]
```diff
@@ -11,11 +11,18 @@ import {
...
+  const value = setting.value ?? setting.defaultValue;
+  const metabotImage = value ? "metabot-happy" : "metabot-sad";
+
   const toggleId = useUniqueId("show-metabot-switch");
   return (
     <MetabotSettingWidgetRoot>
       <MetabotContainer>
-        <MetabotLogo />
+        <img
+          src={`app/assets/img/${metabotImage}.gif`}
```
[the remark this answers is not in the document]
Comment (alxnddr) [reply enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabotSettingWidget/MetabotSettingWidget.tsx:22]: This gif is explicitly set to show animation only once that's why it does not looping forever 🙂

[the remark this answers is not in the document]
Comment (alxnddr) [reply enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabotSettingWidget/MetabotSettingWidget.tsx:22]: Got it now, thanks. Empirically, seems like all browsers keep the previous image, I checked on chromium, webkit, and gecko browsers

[the remark this answers is not in the document]
Comment (WiNloSt) [reply enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabotSettingWidget/MetabotSettingWidget.tsx:22]: Got it. Thanks for testing that out!

**Pull request merged**